spec.bs

```diff
@@ -461,6 +473,146 @@ An <dfn>internal observer</dfn> is a [=struct=] with the following [=struct/item
   [[#promise-returning-operators]] that make use of this, for example.</p>
 </div>
 
+<div algorithm>
+  To <dfn for=Observable>convert to an Observable</dfn> given an {{any}} |value|, run these steps:
+
+  Note: We split this algorithm out from the Web IDL {{Observable/from()}} method, so that
+  spec prose can <a for=Observable lt="convert to an observable">convert</a> an {{Observable}}
+  without going through the Web IDL bindings.
+
+  Note: The resolution of value to its descrete types happens before
+  [=Observable/subscribe callback=] is called. This means mutations of values, such as adding
+  the iterable protocols to the object, will not take affect between the creation of the returned
+  observable, and when it is subscribed to.
```

Comment:
I find this note a tiny bit confusing. We're saying both that:
 1. Narrowly-speaking, adding the iterable protocol to an already-converted object "will not take affect" (spelling). I guess this means that:
    ```js
    const promise = Promise.resolve(10);
    const o = Observable.from(promise);
    promise[Symbol.iterator] = () => {../*Produce the value 11*/};
    o.subscribe(); // Emits value 10.
    ```
 2. Broadly-speaking, "mutations of values" ... "will not take affect". This is a much stronger statement that I'm not comfortable making. In Chromium, we might end up taking the option (2) path described in https://github.com/WICG/observable/issues/127, which means the following mutation would be web-observable in between conversion and subscription:
    ```js
    const iterable = {
      [Symbol.iterator]() { /* ... */ },
    };
    const o = Observable.from(iterable);
    iterable[Symbol.iterator] = () => {throw new Error('foo');}
    o.subscribe(); // Throws error
    ```
The latter is tested in https://github.com/web-platform-tests/wpt/blob/master/dom/observable/tentative/observable-from.any.js#L164-L176 for now, and is easiest to implement in Chromium. So I think I'll either amend or remove this note entirely.

Reply:
Removed in https://github.com/WICG/observable/pull/160/commits/06a1a66575808372a57a83be737af4b8aa61e025.